mandatory/3-playing-computer.js

```diff
@@ -6,13 +6,13 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). // b is not declared/defined
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? // numbers
+  4. How many times is "f1" called? // five times
```

Comment:
The loop executes 5 times, but there is a conditional inside the loop using the % operator. 
Considering that conditional, how many times do you think functions f1 and f2 are called now?